README.md

```diff
@@ -68,22 +76,35 @@ Plural's server side is written in elixir, and exposes a graphql api. The fronte
 
 
 ### Developing Web
-To begin developing the web app, install npm & yarn, then run:
+To begin developing the web app, install [node](https://nodejs.org/en/download/) & [yarn](https://classic.yarnpkg.com/en/docs/getting-started/), then run:
 
 ```sh
-cd assets && yarn install && cd -
+cd www && yarn install && cd ..
 make web
 ```
 
+In chrome, you may get a warning saying "Your connection is not private". To resolve it, enable the `chrome://flags/#allow-insecure-localhost` flag and restart your browser.
+
 ### Developing Server
 
-To make changes to the server codebase, you'll want to install elixir on your machine.  For mac desktops, we do this via asdf, which can be done simply at the root of the repo like so:
+To make changes to the server codebase, you'll want to [install elixir](https://elixir-lang.org/install.html) on your machine. For mac desktops, we do this via [asdf](https://asdf-vm.com/guide/getting-started.html), which can be done simply at the root of the repo like so:
 
 ```sh
 asdf install
 ```
 
-Once elixir is available, all server dependencies are managed via docker-compose, and tests can be run via `mix`, like so:
+<!-- >
+  Remove this line if irrelevant in the future
+</!-->
+In case you're running into this error: `configure: error: cannot find required auxiliary files: install-sh config.guess config.sub` you may consider this [GitHub issue](https://github.com/asdf-vm/asdf-erlang/issues/195#issuecomment-815999279)
```

Comment:
Error --> Google --> Solution kind of fix, but seems hacky. I suggest we find the ground truth on this o ne before merging the PR.

Reply:
I think this is fine for now, elixir never installs smoothly unfortunately